tools/ui-components/src/index.ts

```diff
@@ -1,6 +1,6 @@
 // Use this file as the entry point for component export
 // export { Button } from './button';
-// export { Alert } from './alert';
+export { Alert } from './alert';
```

Comment:
Earlier I tried to export them like this and imported them like you suggested but it gives me the error `Module "@freecodecamp/ui" has no exported member 'CloseButton'.` 

I guess this is because export for `CloseButton` and `AlertProps` is not present in the file `/tools/ui-components/types/src/index.d.ts` but the `types` directory is added in gitignore so even if I made the changes there. It want be part of commit and may generate errors later.

So, Can you tell what should I do ?

Reply:
when you edit the files in the library, it doesn't update the imported file that are build in the `dist`, you need to build it again. Running `pnpm run build:components-library` or `pnpm run develop` in the client directory should do the trick.